draft-ietf-mls-protocol.md

```diff
@@ -344,23 +344,23 @@ the SP provides the following services:
 The goal of this protocol is to allow a group of clients to exchange
 confidential and authenticated messages. It does so by deriving a sequence
 of secrets and keys known only to members. Those should be secret against an
-active network adversary and should have both forward secrecy and
+active network adversary and should hold both forward secrecy and
 post-compromise security with respect to compromise of any members.
 
 We describe the information stored by each client as _state_, which includes
 both public and private data. An initial state is set up by a group creator,
 which is a group containing only itself. The creator then sends _Add_
 proposals for each client in the initial set of members, followed by a _Commit_
 message which incorporates all of the _Adds_ into the group state. Finally, the
-group creator generates a _Welcome_ message corresponding to the Commit and
+group creator generates a _Welcome_ message corresponding to the _Commit_, and
 sends this directly to all the new members, who can use the information
 it contains to set up their own group state and derive a shared
-secret. Members exchange Commit messages for post-compromise security, to add new
-members, and to remove existing members. These messages produce new shared
-secrets which are causally linked to their predecessors, forming a logical
-Directed Acyclic Graph (DAG) of states.
+secret. Members exchange _Commit_ messages for post-compromise security, to
+add new members, and to remove existing members. These messages produce
+new shared secrets which are causally linked to their predecessors, forming
+a logical Directed Acyclic Graph (DAG) of states.
```

Comment:
Wrapping text pollutes "git blame" unnecessarily. We don't have any real line length